MemberDetailsFrontEnd.php

```diff
@@ -0,0 +1,660 @@
+      </div>
+      <div style="float:right;">
+        <!-- <button type="button" name="add" id="add" class="btn btn-success add">Add Member</button> -->
+        <!-- <button type="button" id="" data-toggle="modal" data-target="#add_modal" class="btn btn-success btn-sm ">Add Member</button> -->
+        <button type="button" class="btn btn-success addMember">Add Member</button>
+        <!-- <button type="button" class="btn btn-danger addbadge">Add Badge</button> -->
+      </div>
+    </div>    
+      <div style="clear:both"></div>
+   </div>
+   
+    <br />
+    <div id="alert_message"></div>
+    <table id="user_data" class="table table-bordered table-striped">
+     <thead>
+     <tr>
+       <th width="0%">Member ID</th>
+       <th width="13%">Member Name</th>
+       <th width="10%">Current Color</th>
+       <th width="15%">Member Email</th>
+       <th width="12%">Member Contact</th>       
+       <th width="8%" style="border-right:none;">Action</th>
+       <th width="8%" style="border-right:none;"></th>
+       <th width="6%"></th>
+       <p id="msg" style="display:none">Saved</p>
+      </tr>
+     </thead>
+    </form>
+    </table>
+    
+   </div>
+  </div>
+  
+ </body>
+</html>
+
+
+
+<?php
+include('modal/viewmodal.php');
+include('modal/updatemodal.php');
+include('modal/addmembermodal.php');
+include('modal/addbadgemodal.php');
+?>
+
+<script type="text/javascript" language="javascript" >
+
+$(document).on('click', '.addbadge', function(){  
+      var user_id = $(this).attr("id");  
+      $.ajax({  
+          url:"addbadgefrontend.php",  
+          method:"POST",  
+          data:{user_id:user_id},  
+          dataType:"json",  
+          success:function(data){  
+            response = true;
+                $('#Member').val(data.MemberID); 
+                $('#Section').val(data.SectionID);  
+                $('#Color').val(data.ColorID);
+                $('.modal-title').text("Add Badge");
+                $('#user').val(user_id);
+                $('#action').val("Addme");
+                $('#operation').val("Addme");
+                $('#action').removeClass('btn btn-success').addClass('btn btn-warning'); 
+                $('#AddBadgeModal').modal('show'); 
+                $('#user_data').DataTable().ajax.reload(); 
+          }  
+      });  
+}); 
+
+$(document).on('submit', '#add_badge_form', function(event){
+  $('select').removeAttr('disabled');
+event.preventDefault();
+var user_id = $(this).attr("id");
+ $.ajax({
+  url:"addbadgebackend.php",
+  method:'POST',
+  data:new FormData(this),
+  contentType:false,
+  processData:false,
+  beforeSend:function(){  
+      $('#action').removeClass('btn btn-success').addClass('btn btn-warning');
+      $('#action').val("Adding..");  
+  },
+  success:function(data)
+  {
+   alert(data);
+   $('#add_badge_form')[0].reset();
+   $('#AddBadgeModal').modal('hide');
+   $('#user_data').DataTable().ajax.reload(); 
+   location.reload();

```

Comment:
...and the page refreshes?

Reply:
yes, the page refreshed, but the latest record still didn't display.  It will only display after i INSERTED another record